6.4_remember_remember.py

```diff
@@ -0,0 +1,32 @@
+from PIL import Image
+import os.path
+
+MY_PATH = "resources//code.png"
+
+
+def decryption(path):
+    """
+    Gets a path of an image and decodes a code hidden in pixels
+    :param path: path of image
+    :return: code 
+    """
+    if os.path.exists(path):
+        try:
+            im = Image.open(path)
```

Comment:
replace 'im' Variable names - https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing

Reply:
I changed im to image